io/include/detray/io/backend/detail/grid_reader.hpp

```diff
@@ -209,14 +245,24 @@ class grid_reader {
             detector_builder<typename detector_t::metadata, volume_builder>
                 &det_builder) {
 
+        using grid_type = grid_payload<content_t>::grid_type;
```

Comment:
I think, I should not have named it ```grid_type``` in the payload, that's a bit confusing
```suggestion
        using grid_io_id_t = grid_payload<content_t>::grid_type;
```